src/game_inventory.cpp

```diff
@@ -401,6 +401,24 @@ item_location game::inv_map_splice( const item_filter &filter, const std::string
                          title, radius, none_message );
 }
 
+class liquid_inventory_filter_preset : public inventory_filter_preset
+{
+    public:
+        explicit liquid_inventory_filter_preset( const item_location_filter &filter ) :
+            inventory_filter_preset( filter ) {
+            append_cell( []( const item_location & loc ) {
+                if( loc.get_item() ) {
+                    units::volume vol = loc.get_item()->max_containable_volume();
+                    return string_format( "%.2f L", units::to_liter( vol ) );
```

Comment:
Please use `vol_to_string` to respect the volume unit setting. (and `volume_units_abbr` for the unit).

Reply:
should be changed in the update